charts/pega/charts/constellation/values.yaml

```diff
@@ -1,12 +1,8 @@
 ---
 # Parameters for Constellation operation:
 # Docker repos and tag for image
-dockerRepos: local/constellationui/service:v2
-# dockerRepos: meshbincam.pega.com:7000/constellationui/service
-# dockerRepos: YOUR_IMAGE_HERE
-# Url (protocol+host) for Infinity cluster - used for REST calls
-# infinityUrl: http://1.2.3.4:5678/prweb/
-infinityUrl: http://web.mypega.example.com:30080/prweb/
+dockerRepos: meshbincam.pega.com:7000/constellationui/service
+infinityUrl: http://pega-web
```

Comment:
I still do not think we should need an explicit parameter to define the infinityUrl since the default value can be derived from the tier name from the pega deployment.  Is there a case where you would be deploying constellation along side an existing (already deployed) infinity environment?